Automattic/wp-calypso
# Make the `public/` directory disposable.

This simplifies the build process, allowing us to drop `public/` entirely during clean.

#### Changes proposed in this Pull Request

* Rename repo `public/` to `static/`.
* Copy files from `static/` to `public/` during build.
* Delete `public/` entirely when cleaning.
* Update `.gitignore` to ignore all of `public/`.

#### Testing instructions

* Ensure `npm run distclean` works
* Ensure `npm run start` works
* Ensure build completes normally and there are no missing resources
